pkg/resources/resource_cluster.go

```diff
@@ -174,7 +173,8 @@ func clusterRead(ctx context.Context, d *schema.ResourceData, meta interface{})
 		return diag.FromErr(err)
 	}
 
-	if err := d.Set("disk", s.Disk.Bool); err != nil {
+	// The disk attr is deprecated and is not configurable
+	if err := d.Set("disk", true); err != nil {
 		return diag.FromErr(err)
 	}
```

Comment:
Do we need to set this at all?

Reply:
I think we still need to keep this around for backward compatibility, otherwise it might break existing projects. That said, we can clean it up once we decide to do a minor version bump.